src/pages/using-signalen.md

```diff
@@ -29,3 +21,21 @@ Dan kun je op de volgende manieren bijdragen:
 Iedere twee weken komt de Signalen product steering groep bij elkaar om alle nieuwe feature requests met elkaar te bespreken en te beslissen welke features verder ontwikkeld worden. Als je feature request in behandeling wordt genomen ben je van harte welkom om in het overleg jouw feature request nader toe te lichten.
 
 Wil je deelnemen in de product steering group? Of meer weten hoe besluiten in de Signalen community genomen worden neem dan [contact](/contact/) met ons op.
+
+## Voor ontwikkelaars
+
+Ben je een ontwikkelaar en wil je ons helpen met de doorontwikkeling van Signalen? Neem dan contact met ons op via de [Common Ground Slack](https://join.slack.com/t/samenorganiseren/shared_invite/zt-dex1d7sk-wy11sKYWCF0qQYjJHSMW5Q) om te verkennen hoe we samen Signalen beter kunnen maken!
+
+- Kijk [hier](https://github.com/signalen) voor de Signalen-repository op Github
+- [Gebruikersdocumentatie en implementatiegids](https://docs.signalen.org)
+- [Signalen API documentatie](https://api.data.amsterdam.nl/api/swagger/?url=/signals/swagger/openapi.yaml#/default/get_signals_v1_private_signals_geography)
```

Comment:
let's remove the '#/default/get_signals_v1_private_signals_geography' from this URL

Reply:
good catch! removed it